.env

```diff
@@ -29,6 +29,12 @@ WEB_HOST=web
 SELENIUM_HOST=selenium
 
 # Enable recommendations by setting valid id, key and uri
+# In order for this to work you also need to have the EzSystemsRecommendationBundle installed
```

Comment:
this should have been in ezplatform to avoid conflicts

Reply:
ok, removed